app/assets/javascripts/TortoiseJS/agent/linkdrawer.coffee

```diff
@@ -95,12 +95,13 @@ class window.LinkDrawer
     baseThickness  = 3 / 2
 
     if thickness <= 1
-      scale         = 1 / @view.onePixel / 2
+      scale         = 1 / @view.onePixel / 7
       realThickness = thickness * baseThickness
     else
       scale         = thicknessFactor / 2
       realThickness = baseThickness
 
+    ctx.lineWidth = 10
```

Comment:
I'm not a big fan of setting the context width directly here, since that looks like it should be the job of `@linkShapeDrawer`, and that's why we're going to the trouble of calculating a `realThickness` value and passing it in.  

I tried setting a different `realThickness` value first by multiplying a constant on line 99, then by passing in hard-coded values directly, and didn't see a difference in link arrow size.  Turns out that because the `ShapeDrawer` takes in a `@onePixel` field in [the constructor](https://github.com/NetLogo/Galapagos/blob/staging/app/assets/javascripts/TortoiseJS/agent/drawshape.coffee#L4), but no value is being passed in [here](https://github.com/NetLogo/Galapagos/blob/staging/app/assets/javascripts/TortoiseJS/agent/linkdrawer.coffee#L14).  That left a default line width value being used, which is why it can appear so thin.

So I think I'd like to see the following modifications: 

- Remove the direct setting of the context `lineWidth`.
- Fix the existing bug where the `@view.onePixel` value is not passed to the `@lineShapeDrawer` constructor, so it'll actually use the `realThickness` value that is passed in. 
- Add a minimum value check for the `realThickness` value (say, `0.5`) before it's passed to the `@linkShapeDrawer` to draw the link shape.

Reply:
@LaCuneta Okay. I will try this and get back to you.